vite.config.ts

```diff
@@ -26,19 +49,17 @@ export default defineConfig({
       entry: resolve('src', 'index.ts'),
       name: 'ChatAiWidget',
       formats: ['es', 'umd'],
-      fileName: (format) => `index.${format}.js`
+      fileName: (format) => `index.${format}.js`,
     },
     rollupOptions: {
-      external: [
-        'react',
-        'react-dom',
-      ],
+      plugins: [terser()],
```

Comment:
how much in % is size reduced by terser?

Reply:
- With
<img width="490" alt="Screenshot 2024-04-30 at 3 44 31 PM" src="https://github.com/sendbird/chat-ai-widget/assets/10060731/ff25bfa1-2205-4b45-938d-2991ca5a74de">

- Without
<img width="504" alt="Screenshot 2024-04-30 at 3 44 05 PM" src="https://github.com/sendbird/chat-ai-widget/assets/10060731/6f630eca-8d52-422a-88fc-d95f2419841a">

Reply:
oh it is less than what I expected (>= 20%)